Jenkinsfile

```diff
@@ -111,7 +111,9 @@ def runIntegrationTest(String description, String kubeprodArgs, String ginkgoArg
                     sh "kubectl --namespace kube-system get po,deploy,svc,ing"
 
                     try {
-                        sh "./bin/kubeprod -v=1 install ${kubeprodArgs} --manifests=manifests"
+                        retry(3) {
```

Comment:
Instead, is it sufficient to run `kubectl api-versions` somewhere above - and loop/delay until that succeeds?  Hopefully that will delay long enough that whatever `kubeprod install` is hitting will have stabilised.  Again, I would rather we didn't treat `kubeprod install` specially at all, because flakiness in this command is flakiness that the user will see (and we should instead fix `kubeprod install`).

Reply:
hmm.. that makes sense. i'll make this change